package.json

```
@@ -31,10 +31,10 @@
     "generate:docs": "yarn workspace docs generate:docspage",
     "generate:component": "yarn workspace docs generate:component",
     "run-all": "wt -d packages/ui --title vuestic-book cmd /k yarn serve --port 8080; split-pane -d packages/docs --title vuestic-docs cmd /k yarn serve --port 8081",
-    "release:montly": "cd packages/ui && tsx build/bump-version.ts --type=monthly",
-    "release:weekly": "cd packages/ui && tsx build/bump-version.ts --type=weekly",
-    "release:next": "cd packages/ui && tsx build/bump-version.ts --type=next",
-    "release:experimental": "cd packages/ui && tsx build/bump-version.ts --type=experimental"
+    "release:montly": "cd packages/ui && tsx scripts/release-script.ts --release-type=monthly",
```

This scripts must be executed with npm, but not yarn (at least on my PC with Ubuntu WSL.)

Like
```sh
npm run release:monthly
```
Make sure to write it in notion or some other docs.

Oh, and also typo:
```suggestion
    "release:monthly": "cd packages/ui && tsx scripts/release-script.ts --release-type=monthly",
```

---

> PC with Ubuntu WSL

Yeah, windows is not great for that :<.

Perhaps workspaces feature would work out here. Maybe you can test it?